pypeit/tests/test_load_images.py

```diff
@@ -183,6 +183,15 @@ def test_load_goodman():
     except:
         pytest.fail('Bok BC test data section failed: {0}'.format(ifile))
 
+@dev_suite_required
```

Comment:
Nice!

Reply:
Are there other tests I should add as part of integrating LDT/DeVeny into PypeIt?

Reply:
Nah.  Thanks for asking!